prow/plugins/config.go

```diff
@@ -62,7 +62,7 @@ type Configuration struct {
 	ConfigUpdater              ConfigUpdater          `json:"config_updater,omitempty"`
 	Golint                     Golint                 `json:"golint"`
 	Heart                      Heart                  `json:"heart,omitempty"`
-	Label                      *Label                 `json:"label,omitempty"`
+	Label                      Label                  `json:"label"`
```

Comment:
Does it make sense to remove the `omitempty` here? I mean, if you don't specify it in json, you will get a default struct, won't you?

Reply:
It is technically used also when writing out, if it's empty and `omitempty` it won't get written when you serialize a config. but I am not aware of anyone doing this? Could keep it to be safe? @cjwagner @fejta